apps/admin_app/lib/admin_app_web/templates/product/form.html.eex

```diff
@@ -188,7 +188,7 @@
       <% end %>
     </div>
    <% else %>
-    <%= if !is_child_product(@parent_product) do %>
+    <%= if !Product.is_child_product(@parent_product) do %>
```

Comment:
@jyotigautam can we use `unless` here instead of `if !Product...` that would more succinct.

Reply:
thanks for this.